HostSupport/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ set(HostSupport_SOURCES
 add_library(HostSupport STATIC ${HostSupport_SOURCES})
 target_link_libraries(HostSupport
     PRIVATE
-        expat::expat
+        expat
```

Comment:
this may be an error, I'll check if this is required

Reply:
required for me, else I get:
```
-- Configuring done
CMake Error at HostSupport/CMakeLists.txt:44 (target_link_libraries):
  Target "HostSupport" links to:

    expat::expat

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

Reply:
Please check your version of expat and its CMake config files. Current versions of expat (starting from `2.2.8`) use the `expat::` CMake namespace.

https://github.com/libexpat/libexpat/blob/R_2_2_8/expat/CMakeLists.txt#L505-L512

Reply:
cmake 3.24.0 + expat 2.4.8

Reply:
cmake bundles a more official way of finding expat https://cmake.org/cmake/help/latest/module/FindEXPAT.html
note that it's "Expat", not "expat", and remember that most macOS (and windows) filesystems are case-insensitive, thus this one gets used, not the one installed with expat

Reply:
I pushed the change [1e460f5](https://github.com/NatronGitHub/Natron/pull/834/commits/1e460f50935002d7acccf6eba062057be0714271) so that it works on any system.

I moved the find_package to HostSupport, since it's the only module that uses expat (same should be done with ceres, which should go to Engine/CMakeLists.txt)